### ISSUE TITLE
Format long ERB lines

### ISSUE THREAD

**Issue by lcampanari** [nebulab/erb-formatter#3]
Hey, thanks for this gem!

The readme says "attempts to limit length (configurable)". How can I configure that? 

I tried running the following case in the command line

https://github.com/nebulab/erb-formatter/blob/a710ec2d26bdac49f6e63030fe893f103e3c7961/test/erb/test_formatter.rb#L126-L136

with

```
echo "<div> <%=render MyComponent.new(foo:barbarbarbarbarbarbarbar,bar:bazbazbazbazbazbazbazbaz, baz:bahbahbahbahbahbahbahbabahbahbahbah, bah: fozfozfozfozfozfozfozfofzofzofzofozfozfozofz)%> </div>" | erb-format --stdin
```

and the result was

```
<div>
  <%= render MyComponent.new(foo:barbarbarbarbarbarbarbar,bar:bazbazbazbazbazbazbazbaz, baz:bahbahbahbahbahbahbahbabahbahbahbah, bah: fozfozfozfozfozfozfozfofzofzofzofozfozfozofz) %>
</div>
```

Is there anything I need to do in order to format long embedded ruby lines?

**Comment by elia** [comment]
For now it can only be passed when used directly in Ruby (see below), but ideally there should be a CLI flag to set that number.
Happy to review a PR if you want to take up the challenge 🙌 

https://github.com/nebulab/erb-formatter/blob/main/lib/erb/formatter.rb#L77

**Comment by cj** [comment]
@elia I seem to be having an issue with 0.4.1 and long lines. I am doing the following `bundle exec era-formatter % --print-width 120` and the result is:

<img width="876" alt="image" src="https://user-images.githubusercontent.com/1819/214645437-b75d8f2d-00b7-406d-b9ec-dedf249d5f79.png">

I would have expected the result to be:

<img width="893" alt="image" src="https://user-images.githubusercontent.com/1819/214645540-dc8cba2e-f422-489a-8c66-b81019ef3727.png">

As the line length of the longest line is only 99.

**Comment by elia** [comment]
@cj just released v0.4.2, let me know if it doesn't work and we'll reopen 👍

**Comment by cj** [comment]
@elia works perfectly, thank you so much!
